packages/bbui/src/helpers.js

```diff
@@ -106,3 +106,29 @@ export const deepSet = (obj, key, value) => {
 export const cloneDeep = obj => {
   return JSON.parse(JSON.stringify(obj))
 }
+
+/**
+ * Copies a value to the clipboard
+ * @param value the value to copy
+ */
+export const copyToClipboard = value => {
```

Comment:
Never seen this implementation of copy to clipboard before. Love the textarea workaround if the browser API isn't available 👌